__tests__/data/mautic.json

```diff
@@ -0,0 +1,842 @@
+[
+    
+  {
+    "description": "Create a new Conatct with address as String",
+    "input": {
+      "message": {
+        "anonymousId": "anon-id-new",
+        "context": {
+          "ip": "14.5.67.21",
+          "library": {
+            "name": "http"
+          },
+          "traits": {
+            "firstName": "Anant",
+            "lastName": "jain",
+            "role": "Manager",
+            "address": "Flat No 58 ABC building XYZ Area near PQRS , 354408",
+            "hasPurchased": "yes",
+            "email": "abc@xyz.com",
+            "title": "Mr",
+            "phone": "9876543212",
+            "state": "Uttar Pradesh",
+            "zipcode": "243001",
```

Comment:
As I see all your test cases have actual credentials, which should not be the case. So mock the axios calls and use dummy username passwords. Applicable for all the test cases added below.